src/script/lua_api/l_mainmenu.cpp

```diff
@@ -577,6 +577,13 @@ int ModApiMainMenu::l_get_favorites(lua_State *L)
 			lua_settable(L, top_lvl2);
 		}
 
+		float ping = servers[i]["ping"].asFloat();
+		if (ping) {
```

Comment:
I wasn't sure about it. According to my readings, the variable will be uninitialized (undefined behavior) since it's a local variable.